solr_config/managed-schema

```diff
@@ -447,25 +447,14 @@
       <tokenizer class="solr.WhitespaceTokenizerFactory"/>
     </analyzer>
   </fieldType>
-  <fieldType name="myURL" class="solr.TextField" positionIncrementGap="100">
-   <analyzer type="index">
-    <tokenizer class="solr.StandardTokenizerFactory"/>
-    <filter class="solr.LowerCaseFilterFactory"/>
-   </analyzer>
-    <analyzer type="query">
-    <tokenizer class="solr.KeywordTokenizerFactory"/>
-    <filter class="org.example.filters.ResolveUrlTokenFilterFactory"/>
-    </analyzer>
-  </fieldType>
   <field name="_nest_path_" type="_nest_path_"/>
   <field name="_root_" type="string" docValues="false" indexed="true" stored="false"/>
   <field name="_text_" type="text_general" multiValued="true" indexed="true" stored="false"/>
   <field name="_version_" type="plong" indexed="false" stored="false"/>
-  <field name="directed_by" type="text_general"/>
-  <field name="genre" type="text_general"/>
-  <field name="id" type="string" multiValued="false" indexed="true" required="true" stored="true"/>
-  <field name="initial_release_date" type="pdate" stored="true"/>
-  <field name="name" type="text_general" multiValued="false" stored="true"/>
+  <field name="content" type="text_general" uninvertible="true" indexed="true" stored="false" multiValued="false"/>
```

Comment:
Do we need that uninvertible attribute here?

Reply:
fixed